change/@ni-nimble-components-e3d5df53-15c3-4fe1-8552-ad4db8dbddbd.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc the chromatic stories don't seem to reflect the source changes. Reminder about https://github.com/ni/nimble/issues/2118